build/webcomponents/js/field-category/field-category.js

```diff
@@ -0,0 +1,35 @@
+customElements.define('joomla-field-category', class extends HTMLElement {
+  constructor() {
+    super();
+
+    this.element = '';
+    Joomla.loadingLayer('load', document.body);
+  }
+
+  connectedCallback() {
+    // Check if custom fields are enabled
+    if (this.getAttribute('custom-fields-enabled') !== 'true') {
```

Comment:
What is the exact reason for this attribute?

Reply:
custom fields (the component) can be disabled

Reply:
Hm. And what should happen when its disabled? According to https://github.com/joomla-projects/joomla-es6/pull/75/files#diff-4d276fa5ac7a065127d4620092ed38fdR328 it is a static value and i dont understand (yet) why we need this in client side code? In my simple world the ce should not render at all, when custom fields (the component) is disabled.

Reply:
Ok you’re right, but when I started this I had in mind the complete replacement of both functions there in one element(eg custom fields and create new category) and in such a scenario controlling the element through an attribute was making sense

Reply:
Ok, can we remove it for now and add it when we need it?